tekton/integration_pipeline.go

```diff
@@ -331,6 +331,10 @@ func (r *IntegrationPipelineRun) WithSnapshot(snapshot *applicationapiv1alpha1.S
 		_ = metadata.CopyLabelsByPrefix(&snapshot.ObjectMeta, &r.ObjectMeta, prefix)
 	}
 
+	if metadata.HasAnnotation(r, PipelinesAsCodePrefix+"/log-url") {
+		_ = metadata.SetAnnotation(r, PipelinesAsCodePrefix+"/log-url", r.Name)
```

Comment:
I think the value of the annotation should be more like url
`https://console.redhat.com/preview/application-pipeline/ns/rhtap-shared-team-tenant/pipelinerun/rhtap-cli-on-pull-request-9rrhz` but is replaced by the integration plr name. HDYT?

Reply:
all right, 
can I call the function from status/format  [FormatPipelineRunURL](https://github.com/konflux-ci/integration-service/blob/1025b17955efd24a334e549bba327db7c72ad0e5/status/format.go#L248)  or I shall create different one ?